packages/taboule/public/index.html

```diff
@@ -7,26 +7,30 @@
       #taboule-query-value {
         margin: 0 20px 30px 0;
       }
+      #taboule-reload {
+        margin-left: 20px;
+        cursor: pointer;
+      }
     </style>
   </head>
   <body>
     <div>
       <h2>Taboule</h2>
       <div>
         <select id="taboule-query-value">
-          <option value="ccRelatedUsers">Content creator Related Users</option>
-          <option value="personalSearches">Personal Searches</option>
-          <option value="personalVideos">Personal Videos</option>
-          <option value="personalHomes">Personal Homes</option>
-          <option value="personalAds">Personal ADS</option>
-          <option value="getExperimentById">Compare Experiment</option>
-          <option value="getExperimentList">Experiment list</option>
+          <option value="ccRelatedUsers">
+            YCAI Content creator Related Users
+          </option>
+          <option value="personalSearches">Youtube Personal Searches</option>
```

Comment:
Can you add a prefix also to the query value? So it's more explicative when we use it in the websites